### ISSUE TITLE
Batch question

### ISSUE THREAD

**Issue by jimdubbs** [cyraxx/pogobuf#42]
When using the batch functionality, is there any limit to the amount/types of requests that can be made? For example if I have 10 gyms I want to pull the location of I need to first do a heartbeat at the location, then getGymDetails(). Would it be better to batch all of them together or not? Currently I just have several clients, and when looping through the gym collection pull a random one to do the heartbeat/getGymDetails.

Basically Im just wondering if anyone has any insight on the most effecient way to make these sort of calls.

**Comment by cyraxx** [comment]
On the library's side there is no limit to how many requests can go into one batch call. What the server is going to do with it is a different question that we don't know the answer to.

However, the RPC envelope (that contains all the requests in one batch call) includes the player's latitude and longitude. So if you're changing location it would probably be better to make separate calls.
